server/service/src/processors/transfer/invoice/common.rs

```diff
@@ -84,6 +85,7 @@ pub(crate) fn generate_inbound_lines(
                     stock_line_id: None,
                     location_id: None,
                     inventory_adjustment_reason_id: None,
+                    donor_default_id: None,
```

Comment:
Imagine would be easiest just to take the donor id destructured above and pass it through here?